src/onemod/constraints/functions.py

```diff
@@ -3,79 +3,94 @@
 
 from polars import Series
 
-def bounds(ge: float = None, le: float = None) -> Callable[[Series], None]:
+
+def bounds(
+    ge: float | None = None, le: float | None = None
```

Comment:
Really minor point, I think it will be great we use `lb` and `ub` compare to `ge` and `le`, more widely used in our context.

Reply:
Good to know! I have not thought about the constraint functions we have at all, just made them for the sake of testing core validation functionality really, so I would like to follow up with you to pick your brain on what useful constraint functionality we could add to OneMod out of the box.